packages/terra-application/src/page/Page.jsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+
+import { NavigationPromptCheckpoint, getUnsavedChangesPromptOptions } from '../navigation-prompt';
+import useNotificationBanners from '../notification-banner/private/useNotificationBanners';
+import usePagePortal from '../page-container/usePagePortal';
+import { ApplicationIntlContext } from '../application-intl';
+
+import PageHeader from './PageHeader';
+import PageActions from './PageActions';
+import PageAction from './PageAction';
+import PageToolbar from './PageToolbar';
+
+import styles from './Page.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The string description of the Page to present to the user. This value is
+   * also used to build description text for screen readers and accessibility
+   * tools.
+   */
+  label: PropTypes.string.isRequired,
+  /**
+   * An object containing generic data describing the content of the Page.
+   * The metaData will be accessible by components who are notified of Page
+   * presentation changes.
+   */
+  metaData: PropTypes.object,
+  /**
+   * A PageActions instance defining the Actions to present within the Page's
+   * header. Only the PageActions component is supported.
+   */
+  actions: PropTypes.element,
+  /**
+   * A component to present additional controls to the Page user. The provided
+   * component will be rendered below the Page's header.
+   */
+  toolbar: PropTypes.node,
+  /**
+   * A function to be executed upon selection of the Page's back button. If not
+   * provided, the back button will not be rendered.
+   */
+  onRequestClose: PropTypes.func,
+  /**
+   * When true, the Page will not prompt the user prior to executing
+   * `onRequestClose` in the presence of rendered UnsavedChangesPrompts.
+   * Use this prop to customize UnsavedChangesPrompt handling prior to Page
+   * closure.
+   */
+  dangerouslyDisableUnsavedChangesPromptHandling: PropTypes.bool,
```

Comment:
remind me, in what situations does this get used?

Reply:
This is the "I'm handling my Page's unsaved changes differently and don't want two prompts to pop up" prop. I'd anticipate something similar on the new Modal.